edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/util/aggrigation/AggrigationMetric.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Util.Aggregation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+
+    // This class is used to temporaraly compare similar metrics. It doesn't include values, and
+    // the aggrigate tag is removed
+    public class AggrigateMetric
```

Comment:
Still several instances of "aggrigate" hanging around, including folders, filenames, and comments.